src/scenes/CreateEvent/index.js

```diff
@@ -378,8 +398,54 @@ export default class CreateEvent extends Component {
     if (nextProps.txReturn) {
       this.onClose();
     }
+
+    if (!_.isUndefined(nextProps.addressValidated)) {
+      this.renderResultSetter();
+    }
   }
 
+  renderResultSetter = () => {
+    const { intl: { formatMessage } } = this.props;
+    return (
+      <Field
+        required
+        fullWidth
+        name={ID_RESULT_SETTER}
+        placeholder={formatMessage(messages.resultSetterPlaceholder)}
+        component={this.renderResultSetterTextField}
+        validate={[this.validateResultSetterAddress]}
+      />
+    );
+  }
+
+  renderResultSetterTextField = ({ input, placeholder, startAdornmentLabel, meta: { touched, error }, ...custom }) => {
```

Comment:
rename to `resultSetterTextField`. think its wrong to name it `renderThisComponent` and just name it the component itself.